internal/constants/constants.go

```diff
@@ -287,3 +290,9 @@ const SvcAppName = "State Service"
 
 // StateAppName is the name we give our state cli executable
 const StateAppName = "State Tool"
+
+// AutoStartPath is the config key that holds the path to the auto start file
+const AutoStartPath = "auto_start_path"
```

Comment:
atm you're using `AutoStartPath` in platform specific logic to set up the autostart logic, but then in the destroy logic you just assume all platforms use this entry. This doesn't seem necessary, just have create and destroy functions that each platform can implement as they see fit.